src/libraries/System.Formats.Tar/src/System.Formats.Tar.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Please move this back. I know Viktor has been fixing it so all .csprojs have TargetFrameworks as the first Property.

Reply:
Oh, thanks for noticing. I had the merge conflict with main and didn't notice the position.